source/maintain_project/review_content/index.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: Review content regularly
```

Comment:
This title feels a little like it's advice. Would a title like 'Set up regular content reviews' work better?